pydantic/fields.py

```diff
@@ -136,6 +138,7 @@ def Field(
     max_items: int = None,
     min_length: int = None,
     max_length: int = None,
+    allow_mutation: bool = None,
```

Comment:
```suggestion
    allow_mutation: bool = True,
```

It's good to set the correct default here to make the code self-documenting - e.g. many people use the signature hint provided by pycharm or vscode to understand arguments.

Reply:
it'll also simplify the signature below.

Reply:
147 tests fail with that change.  Looks like it's a ValueError with message `the following field constraints are set but not enforced: allow_mutation.`

I'd probably have to add an exception for checking unenforced constraints for allow_mutation because it's default is a True instead of None.  I'll try to dig into a bit and if there is a straightforward solution I'm comfortable with then I'll make a commit.  Suggestions are welcome here.

Reply:
Just made another commit that allows for this by including a check for the default value of the constraint instead of checking only for None.